scripts/commands/archive-list.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/** @var array $args */
+
+use Lindyhopchris\ShoppingList\Application\Commands\ArchiveShoppingList\ArchiveShoppingListModel;
+use Lindyhopchris\ShoppingList\Common\Validation\ValidationException;
+use Lindyhopchris\ShoppingList\Container;
+
+if (1 > count($args)) {
+    echo 'Expecting one argument: shopping list slug.' . PHP_EOL;
+    exit(1);
+}
+
+$command = Container::getInstance()->getArchivedShoppingListCommand();
```

Comment:
Suggestion: just noticing this now, but I think we should rename the method on the container to `getArchiveShoppingListCommand()`, i.e. it shouldn't be past-tense as a command is an "action", rather than something that has happened.